common/analysis/lint_rule_status.cc

```diff
@@ -105,6 +108,36 @@ struct LintViolationWithStatus {
   }
 };
 
+std::string LintStatusFormatter::ReplaceWithHelperTokens(
+    const std::vector<verible::TokenInfo>& tokens, absl::string_view reason,
+    absl::string_view path, absl::string_view base) const {
+  if (tokens.empty()) {
+    return std::string(reason);
+  }
+
+  size_t end_pos = reason.find("@", 0), beg_pos = 0;
```

Comment:
make these two separate lines; it is easy to overlook the beg_pos declaration. Also, in that case you can use `beg_pos` already

```c++
size_t beg_pos = 0;
size_t end_pos = reason.find("@", beg_pos);
```